sensor/common/detector/metrics/metrics.go

```diff
@@ -120,8 +120,61 @@ var (
 		Name:      "detector_network_flows_dropped_total",
 		Help:      "A counter of the total number of network flows that were dropped if the detector buffer was full",
 	})
+
+	detectorBlockScanCalls = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "block_scan_calls",
+		Help:      "A counter that tracks the operations in blocking scan calls",
+	}, []string{"Operation", "Path"})
+
+	scanCallDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "scan_call_duration",
+		Help:      "Time taken to call scan in milliseconds",
+		Buckets:   prometheus.ExponentialBuckets(4, 2, 8),
```

Comment:
Let's do more - those are only milliseconds
```suggestion
		Buckets:   prometheus.ExponentialBuckets(10, 100, 1000, 10_000, 60_000, 120_000),
```

Reply:
🤔 Are you suggesting to hardcode those bucket sizes? With the `ExponentialBuckets` we get `4, 8, 16, 32, 64, 128, 256, and 512` if I'm not mistaken. I've bumped the number to 16 to get up to 131.000. Let me know what you think.

Reply:
bump is totally fine. I mistaken that with having only 3 buckets 2,4,8 🤦